### ISSUE TITLE
Allow users to kick people from multicontroller scenarios

### ISSUE THREAD

**Issue by svalencia014** [mmp/vice#246]
It would be helpful to remove people who invade scenarios if you did not setup password protection, instead of having to create a new session with a password

**Comment by mmp** [comment]
Yep, this wouldn't be too hard to add. It probably would make sense to prohibit that IP address from joining that server afterward.

Maybe also we should add a command to set up a password after creating a server.
